content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -120,6 +118,50 @@ The release captain has unlimited power to make changes to the release branch to
 
 Most issues are non-blocking. Fixes to non-blocking issues can be fixed in `main` by the code owner who can then `git cherry-pick` those commits into the release branch with the approval of the release captain. Alternatively, broken features can be reverted out of the release branch or disabled via feature flags if they aren't ready or are too buggy.
 
+## Minor release process
+
+### 1) Start a minor release
+
+Major and minor releases are released on a fixed scheule, see [when we release](#when-we-release).
+
+### 2) Minor release tracking issue
+
+The tracking issue for the current minor release is created as a part of the post-release step from the previous minor release. [3.36.0 release tracking issue] is a good example of a minor release tracking issue.
+
+The Release Captain should review and follow the instruction in the patch release tracking release for next steps. At a high level, it includes the following steps:
+
+- Build a release candidate and verify CI passes
+- Publish final images
+- Update documentation
+- Update references of the image tag to the new version using batch change
+
+### 3) Wrapping up
+
+Follow the instruction from post-release in the release tracking issue to wrap up the release.
+
+## Patch release process
+
+### 1) Kickstart a patch release using the [Sourcegraph release tool]
+
+CE or the products team will start requesting a patch release by submitting a [patch release request]. The patch release to [revert poor onboarding UX change] is a good example.
+
+Upon the Delivery team or the Release Captain receives the patch release request and we have decided to rollout a new patch release, the Release Captain should follow the instruction in the [patch release request] issue to kickstart the patch release process. The instruction is located at the bottom of the [patch release request].
+
+### 2) Patch release tracking issue
+
+The [Sourcegraph release tool] creates a patch release tracking issue which contains a list of action items the Release Captain has to perform. [3.36.2 patch release tracking issue] is a good example.
```

Comment:
We should just link to the template here, instead of a potentially outdated example